doc/api/next_api_changes/behavior/28061-JMK.rst

```diff
@@ -0,0 +1,14 @@
+imshow *interpolation_stage* default changed to 'antialiased'
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The *interpolation_stage* keyword argument `~.Axes.imshow` has a new default
+value 'antialiased'.  For images that are down-sampled or up-sampled less than
+a factor of three, image interpolation will occur in 'rgba' space.  For images
```

Comment:
```suggestion
value 'antialiased'.  For images that are up-sampled less than a factor of 
three or down-sampled , image interpolation will occur in 'rgba' space.  For images
```

I read it as "(down- or up-sampled) by less than a factor of 3" which is not what is meant.